fiftyone/factory/repos/delegated_operation_doc.py

```diff
@@ -62,6 +64,8 @@ def __init__(
 
         # distributed task fields
         self.parent_id = None  # Only on children
+        self.pipeline_index = 0
```

Comment:
do you really need 2 variables? Can't self.pipeline equal the index if its a pipeline or just none if not a pipeline

Reply:
pipeline index is the current stage of operation that it's on.
pipeline is the definition of the stages